monero-outreach-docs/translations/pt-br/PSA/PSA_pt-br_headers.md

```diff
@@ -0,0 +1,36 @@
+# Headers
+
+- Wallet
+  * My Monero is tight - Meu Monero está seguro
+  * And out of sight - E fora de vista
+  * Closed - Fechado
+
+- Exchange
+  * Monero Hero says: - O herói do Monero diz:
+  * Don’t be a Jive Turkey - Não seja cabeça oca

```

Comment:
Yeah, botoserelepe I also did the search when working the Spanish translation and found the same. The translation I used was, "Cabeza Hueca", is the same used netrik, "cabeça oca". I used it because is not basically a translation but a "slang", not sure if you understand what I mean.